onnxruntime/core/optimizer/common_subexpression_elimination.cc

```diff
@@ -350,7 +446,8 @@ Status CommonSubexpressionElimination::ApplyImpl(Graph& graph, bool& modified, i
   // This is the inverse of the above mapping, except that different NodeArgs can belong to the same
   // equivalence class. In that case these NodeArgs will be "merged" into one.
   std::unordered_map<const NodeArg*, const EquivalenceClass*, NodeArgPtrHash, NodeArgPtrEquality> equivalence_classes;
-
+  std::unordered_map<const ONNX_NAMESPACE::TensorProto*, const EquivalenceClass*, TensorProtoHash, TensorProtoEquality> eq;
```

Comment:
Why do we need a separate map? Can't we use `equivalence_classes` for everything?